site/source/docs/api_reference/Filesystem-API.rst

```diff
@@ -77,6 +77,17 @@ This file system lets a program in *node* map directories (via a mount operation
 
 See `this test <https://github.com/emscripten-core/emscripten/blob/main/tests/fs/test_nodefs_rw.c>`_ for an example.
 
+.. _filesystem-api-noderawfs:
+
+NODERAWFS
+------
+
+.. note:: This file system is only for use when running inside :term:`node.js`.
+
+ This is a special backend as it replaces all normal filesystem access with direct Node.js operations, without the need to do `FS.mount()`. The initial working directory will be same as process.cwd() instead of VFS root directory.  Because this mode directly uses Node.js to access the real local filesystem on your OS, the code will not necessarily be portable between OSes - it will be as portable as a Node.js program would be, which means that differences in how the underlying OS handles permissions and errors and so forth may be noticeable.  This has mostly been tested on Linux so far.
+
+See `this test <https://github.com/emscripten-core/emscripten/blob/main/tests/fs/test_noderawfs_nofollow.c>`_ for an example.
```

Comment:
Linking to the same test as above would be better, as it has a lot more detail there. Can also mention the `#ifndef NODERAWFS` block there, which shows the exact difference between NODEFS and NODERAWFS

Reply:
`#ifndef NODERAWFS` means that the code is *not* included in NODERAWFS mode. That is, the benefit of this mode is to not need to write that code. So we should clarify that in a comment, perhaps something like "Without NODERAWFS, you need to mount a directory for NODEFS. With NODERAWFS, that is not needed and you can simply access files immediately."